opensquirrel/circuit.py

```diff
@@ -74,14 +74,14 @@ def from_string(
             cqasm3_string: a cQASM 3 string
             gate_set: an array of gate semantic functions. See default_gates for examples
             gate_aliases: a dictionary of extra gate aliases, mapping strings to functions in the gate set
-            measurement_set: an array of measurement semantic functions. See default_measurements for examples
+            measure_set: an array of measure semantic functions. See default_measures for examples
```

Comment:
Here's one of those cases where I have my doubts. Wouldn't this comment read better as: `measure_set: an array of measurement semantic functions`?

Reply:
I will change it back to measurement. I wasn't sure about the syntax but now that you say so I think you're right.

Reply:
I think both are fine. 'measure semantic functions' are various semantic functions of the measure instruction that use the `Measure` semantic, and 'measurement semantic functions' are various semantic functions of different kinds of measurement operations (either using or not, the `Measure` semantic). Since we have no other sematic for measurement operations than the `Measure` semantic, both descriptions are possible.